app/components/settings/SettingsWindow.tsx

```diff
@@ -339,57 +346,31 @@ export const SettingsWindow = ({ open, onClose }: SettingsProps) => {
+                          <span className="text-bolt-elements-textPrimary">Debug Info</span>
+                          <Switch
+                            className="ml-auto"
                             checked={isDebugEnabled}
-                            onChange={() => setIsDebugEnabled(!isDebugEnabled)}
+                            onCheckedChange={() => setIsDebugEnabled(!isDebugEnabled)}
                           />
-                          <div
-                            className={classNames(
-                              'settings-toggle__track',
-                              isDebugEnabled ? 'settings-toggle__track--enabled' : 'settings-toggle__track--disabled',
-                            )}
-                          ></div>
-                          <div
-                            className={classNames(
-                              'settings-toggle__thumb',
-                              isDebugEnabled ? 'settings-toggle__thumb--enabled' : '',
-                            )}
-                          ></div>
-                        </label>
+                        </div>
                       </div>
-                    </div>
-                  )}
-                  {activeTab === 'features' && (
-                    <div className="p-4 bg-bolt-elements-bg-depth-2 border border-bolt-elements-borderColor rounded-lg">
-                      <h3 className="text-lg font-medium text-bolt-elements-textPrimary mb-4">Experimental Area</h3>
-                      <div className="flex items-center justify-between mb-2">
-                        <span className="text-bolt-elements-textPrimary">Replace with local models</span>
-                        <label className="relative inline-flex items-center cursor-pointer">
-                          <input
-                            type="checkbox"
-                            className="sr-only"
-                            checked={isJustSayEnabled}
-                            onChange={() => setIsJustSayEnabled(!isJustSayEnabled)}
+
+                      <div className="mb-6 border-t border-bolt-elements-borderColor pt-4">
+                        <h3 className="text-lg font-medium text-bolt-elements-textPrimary mb-4">Experimental Features</h3>
+                        <p className="text-sm text-bolt-elements-textSecondary mb-4">
+                          Disclaimer: Experimental features may be unstable and are subject to change.
+                        </p>
+                        <div className="flex items-center justify-between mb-2">
+                          <span className="text-bolt-elements-textPrimary">Enable Local Models</span>
+                          <Switch
+                            className="ml-auto"
+                            checked={isLocalModelsEnabled}
+                            onCheckedChange={() => setIsLocalModelsEnabled(!isLocalModelsEnabled)}
                           />
```

Comment:
you can actually pass the value from the handler

`onCheckedChange={(value) => setIsLocalModelsEnabled(value)}`

Reply:
Thanks ended up moving it to set cookies at the same time.

const handleToggleLocalModels = (enabled: boolean) => {
    setIsLocalModelsEnabled(enabled);
    Cookies.set('isLocalModelsEnabled', String(enabled));
  };